src/Shell/BootstrapShell.php

```diff
@@ -27,7 +27,8 @@ public function install()
     {
         $this->TwbsAssets->installAssets();
         $this->TwbsAssets->copyAssets();
-        $this->Assets->symlink();
+        $this->Assets->remove('BootstrapUI');
```

Comment:
`AssetsTask::remove()` was added in `3.5.12`, so please update `composer.json` accordingly.